sources/packages/web/src/components/common/AnnouncementBanner.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <banner
+    header="System Announcements"
+    :type="BannerTypes.Warning"
+    v-if="relevantAnnouncements && relevantAnnouncements.length > 0"
```

Comment:
The v-if could be simplified as 

`v-if="relevantAnnouncements.length"`

Reply:
updated